doc/src/user_guide.rst

```diff
@@ -1076,6 +1076,14 @@ Parameterised with
 
     ``blocking`` | ``non-blocking``
 
+4. ``region`` --- optional specification of a region to be written,
+   defined via either diametrically opposite vertices of a bounding
+   cuboid, or the name of a domain boundary (in the latter case the
+   solution in all elements that have faces on the domain boundary
+   is written):
+
+    [(x, y, z),(x, y, z)] | *string*
```

Comment:
``* | ..`` where * corresponds to everything.

Reply:
So just to check, you can actually put * and it will dump everything? I thought that to dump everything you just don't specify a region? Or do both work?

Reply:
The default is ``*`` which corresponds to dump everything.